### DISCUSSION TITLE
Email Invite Warning

### DISCUSSION THREAD

**Discussion by dmezh** [outline/outline#1685] [General] [answered]
Should there be a warning when a user tries to sign in with an email that hasn't been invited?

I implemented a quick fix for my own instance:

https://github.com/dmezh/outline/commit/105cfa13068c122711d9e2fa91b01d9b6d69ef5e

**Comment by tommoor** [comment, marked as answer]
You can see by the comment directly below that line that it's purposefully built this way, returning a success/failure allows enumeration of user accounts and would be considered a small security flaw.

I think the ideal solution would be to send an email to the address if they don't have an account letting them know to request an invite. Such an email would need to be protected by a rate limiter though.
